security-analysis/security-analysis-api/src/main/java/com/powsybl/security/condition/AcDcConverterThresholdCondition.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2025, Coreso SA (https://www.coreso.eu/) and TSCNET Services GmbH (https://www.tscnet.eu/)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.security.condition;
+
+/**
+ * Condition on an acdc converter triggered by a threshold
+ *
+ * @author Bertrand Rix {@literal <bertrand.rix at artelys.com>}
+ */
+public class AcDcConverterThresholdCondition extends AbstractThresholdCondition {
+
+    public static final String NAME = "AC_DC_CONVERTER_THRESHOLD_CONDITION";
+
+    // True to target AC side, false for DC side
+    private final boolean acSide;
+    // Terminal number to target
+    private final int terminalNumber;
+
+    public AcDcConverterThresholdCondition(double threshold, AbstractThresholdCondition.ComparisonType type, String equipmentId,
```

Comment:
It's weird to have `ComparisonType` and `Variable` as inner enum of an "Abstract..." class. Maybe they can be moved somewhere else?

Reply:
Those are very specific to threshold condition. It makes sense to have them in the most generic part of the API. But I can put them outside if you prefer.

Reply:
You are right. We can keep it like this.